app/views/admin/suppliers/index.html.haml

```diff
@@ -0,0 +1,38 @@
+.govuk-grid-row
+  .govuk-grid-column-full
+    %h1.govuk-heading-xl
+      Suppliers
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    %form{method: 'get'}
+      %h2.govuk-heading-s
+        Search
+      .ccs-search-form-group
+        %label.govuk-label.govuk-visually-hidden{for: 'search'} Search
+        %input#search.govuk-input{name: 'search', type: 'text', value: params[:search]}
+        %button.govuk-button Search
+
+  .govuk-grid-column-one-third
+
+.govuk-grid-row
+  .govuk-grid-column-full
+    - unless @suppliers.empty?
```

Comment:
Looks like one of the `unless`/`else` combos was missed here. Otherwise this is looking great!